docs/pages/blog/mui-x-v7.md

```diff
@@ -0,0 +1,361 @@
+
+This feature makes charts more interactive and transforms the way users can explore the data, improving the overall user experience with details and other information accessed on demand.
+
+<img alt="Gauge charts" src="/static/blog/mui-x-v7/click-handler.png" width="450" height="416" loading="lazy" />
+
+You can find more information about specific event handlers for each type of chart by visiting their respective overview page:
+
+- [Line charts click events](https://mui.com/x/react-charts/lines/#click-event)
+- [Bar charts click events](https://mui.com/x/react-charts/bars/#click-event)
+- [Pie charts click events](https://mui.com/x/react-charts/pie/#click-event)
+
+### Built-in Grid
+
+The Axis component now features a built-in Grid, offering users a structured layout for easier data analysis and visualization.
+
+<img alt="Charts axis grid" src="/static/blog/mui-x-v7/axis-grid.png" width="1550" height="654" loading="lazy" />
+
+For details on its usage and configuration, please refer to the [Grid section](https://mui.com/x/react-charts/axis/#grid) in the documentation.
+
+### Reference line
+
+The `ChartsReferenceLine` component enhances data visualization, providing users with a clear reference to better understand and analyze key data points.
+
+<img alt="Charts reference line" src="/static/blog/mui-x-v7/charts-reference-line.png" width="1200" height="840" loading="lazy" />
+
+You can find more details on its [documentation](https://mui.com/x/react-charts/axis/#reference-line).
+
+## Date and Time Pickers
+
+The focus has been primarily on developing the highly anticipated DateTimeRangePicker, alongside a significant effort to enhance accessibility in the fields, ensuring that our components are as inclusive and user-friendly as possible.
+
+### Improved accessibility on Pickers Text Field
+
+You can now set the `enableAccessibleFieldDOMStructure` prop to enhance the DOM structure of fields and pickers components.
+
+This improvement addresses accessibility limitations of the traditional `<input />` element by enabling a new DOM structure that effectively manages aria attributes within multiple input sections.
+
+This feature greatly improves the experience for screen reader users, making interactions with date fields and pickers more intuitive and accessible.
+
+```jsx
+<DateField enableAccessibleFieldDOMStructure />
+<DatePicker enableAccessibleFieldDOMStructure />
+```
+
+For more details, please check out the [documentation section](https://mui.com/x/react-date-pickers/fields/#accessible-dom-structure).
+
+### Support for date-fns v3
+
+The Date and Time Pickers now support both versions of v2 and v3 of date-fns, and you can select your desired version from one of the two import lines:
+
+```jsx
+// date-fns v2.x
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+```
+
+```jsx
+// date-fns v3.x
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFnsV3';
+```
```

Comment:
```suggestion
```jsx
// date-fns v2.x
import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';

// date-fns v3.x
import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFnsV3';
```
```

I think we can put these on the same code block